cmd/cli/plugin/cluster/test/cluster_silentmode.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	clitest "github.com/vmware-tanzu/tanzu-framework/pkg/v1/test/cli"
+)
+
+var (
+	createClusterNegTestSilentMode        *clitest.Test
+	deleteClusterNegTestSilentMode        *clitest.Test
+	availableUpgradesGetNegTestSilentMode *clitest.Test
+	credUpdateNegTestSilentMode           *clitest.Test
+	clusterGetTestSilentMode              *clitest.Test
+	clusterKubeconfGetSilentMode          *clitest.Test
+	clusterMHGet                          *clitest.Test
+	clusterNodePoolSetSilentMode          *clitest.Test
+	clusterNodePoolListSilentMode         *clitest.Test
+	clusterNodePoolDeleteSilentMode       *clitest.Test
+	clusterScaleSilentMode                *clitest.Test
+	clusterUpgradeSilentMode              *clitest.Test
+)
+
+var FuncToExecAndValidateStdError = func(t *clitest.Test) error {
+	if err := t.ExecNotContainsStdErrorString(usage); err != nil {
+		return err
+	}
+	return nil
+}
+
+func initClusterSilentModeUsecases() {
+	createClusterNegTestSilentMode = clitest.NewTest("create cluster", "cluster create", FuncToExecAndValidateStdError)
```

Comment:
nit: are we running each of these commands without args and relying on some being failure cases and some not to exercise the usage silencing behavior?
It may be a good idea group the test cases, and to provide some explanation what we are trying to testing here

Reply:
All cases are failure cases only, yes it's to exercise the silentUsage. I have already grouped all silent use cases, I will details in the comments.